lib/pf/Switch/Ruckus/Ruckus_Legacy.pm

```diff
@@ -0,0 +1,80 @@
+package pf::Switch::Ruckus::Ruckus_Legacy;
```

Comment:
Because I copy pasted it and been interrupted by @julsemaan at the same time ;)
Fixaide